projects/ngx-location-picker/src/lib/components/ngx-location-picker.component.html

```diff
@@ -26,14 +26,14 @@
               <a href="#" [class]="{
                 'a-list__content': true,
                 'is-active': ii === highlightedLocationResult,
-              }" (click)="onLocationSelect($event, location)">
+              }" (mousedown)="onLocationSelect($event, location)">
```

Comment:
Dit lijkt me geen goed idee. Wat is de reden dat je dit doet?

Reply:
Het heeft vooral met deze issue te maken:

https://jira.antwerpen.be/plugins/servlet/mobile#issue/GIS-849 


De reden is omdat wanneer je het vasthoud + klikt na 0.5s, dan wordt het niet geregistreerd als een click event en gebeurt er niks.

Reply:
hey @DanielSavin2001 wat je beschrijft is standaard browsergedrag en weegt dus geenszins op tegen het ongewenste gedrag dat je introduceert met deze aanpassing. Ik vrees dat je echt gaat moeten zoeken naar de oorzaak waarom de click niet doorkomt en daar een fix voor gaat moeten schrijven.

Reply:
Hi @TriangleJuice !

Als dit werkelijk standaard gedrag is van de browser, spreken we dan niet gewoon over een "user input error" ipv een code error? Qua code leek het mij enigszins solide!!

Als ik een korte video maak hiervan om het even te tonen, zou dat kunnen helpen?

Reply:
Hey @DanielSavin2001 het klopt inderdaad wat je zegt. Mogelijk is het een user input error en valt er gewoon niets te fixen 😊